erpnext/projects/report/vehicle_service_feedback/vehicle_service_feedback.py

```diff
@@ -113,7 +113,8 @@ def get_conditions(self):
 			conditions.append("""im.item_group in (select name from `tabItem Group`
 				where lft>=%s and rgt<=%s and docstatus<2)""" % (lft, rgt))
 
-		conditions.append("""ro.project_type != 'Internal'""" )
+		conditions.append("""ro.project_type in (select project_type from `tabProject Type`
+			where is_internal != 'Yes')""")
```

Comment:
`select name` instead of select project_type
always use **name** to join tables. name is the Unique ID

Reply:
Better to do `not in` where is_internal = 'Yes'